__tests__/errors.test.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const UsageError = require("../errors").UsageError;
+const onlyInstancesOf = require("../errors").onlyInstancesOf;
+
+describe("onlyInstancesOf", () => {
+  test("catches specified error", () => {
```

Comment:
@rpl wrote simple version of the second test, web-ext test have `makeSureItFails` helper to avoid accidentally passing. Should we implement some simple version of that for the future?